src/popup/pages/Index.vue

```diff
@@ -144,6 +151,18 @@ export default defineComponent({
       return openModal(MODAL_ACCOUNT_IMPORT);
     }
 
+    /**
+     * TMP: for IOS Migration
+     */
+    onMounted(async () => {
+      await watchUntilTruthy(mnemonic);
+      if (mnemonic.value && IS_IOS) {
+        setActiveAccountByGlobalIdx(0);
+        await discoverAccounts();
+        router.push(loginTargetLocation.value);
```

Comment:
I think we should move user only if the discovery process actually created at least one account.

Reply:
What if the user has created this account, not used it yet, but has send the address somewhere and is expecting funds?